### PR TITLE
Added clarity about the need to import-module Clear-PhysicalDeskHealt…

### PR DESCRIPTION
**Pull request by p8reit**
…hData.ps1 before running the script.

This has caused confusion for customers and this simple change will help users run the mitigation easier

### CODE REVIEW

**Comment by illfated** [comment]
As an outsider to this topic & content, wouldn't it be practical to put the PS command inside a PowerShell code block and preface it with something like "Run this command:" ?

Example:
***
"Removing from Pool" is an intent set when **Remove-PhysicalDisk** is called but stored in Health to maintain state and allow recovery if the remove operation fails. You can manually change the OperationalStatus to Healthy with one of the following methods:

- Remove the physical disk from the pool, and then add it back.
- Run the following command:
  ```powershell
  Import-Module Clear-PhysicalDeskHealthData.ps1
  ```
- Run the [Clear-PhysicalDiskHealthData.ps1 script](https://go.microsoft.com/fwlink/?linkid=2034205) to clear the intent. (Available for download as a .TXT file. You'll need to save it as a .PS1 file before you can run it.)
***
(This is purely a guess from my side. For all I know, the current context and flow in this document could very well make it obvious to the users that the added line is intended to be run as a PS command.)

**Comment by eross-msft** [comment]
@jasongerend  Please take a look at this text and let me know if it's okay to merge. Thanks!!

File: WindowsServerDocs/storage/storage-spaces/troubleshooting-storage-spaces.md [hunk 1]
```diff
@@ -415,6 +415,7 @@ In a Windows Server 2016 Storage Spaces Direct cluster, you might see the Health
 "Removing from Pool" is an intent set when **Remove-PhysicalDisk** is called but stored in Health to maintain state and allow recovery if the remove operation fails. You can manually change the OperationalStatus to Healthy with one of the following methods:
 
 - Remove the physical disk from the pool, and then add it back.
+- Import-Module Clear-PhysicalDeskHealthData.ps1
```
Comment (illfated) [inline WindowsServerDocs/storage/storage-spaces/troubleshooting-storage-spaces.md:418]: ❗ Typo warning: -Desk- should be **Disk** in this file name.
```suggestion
- Import-Module Clear-PhysicalDiskHealthData.ps1
```
See also the related issue ticket #3309 (**Clear-PhysicalDiskHealthData.ps1 - make sure you import-module first**) for details.

**Comment by illfated** [comment]
This PR should close issue ticket #3309 when merged (can be automated by adding the following below the existing text in the main PR comment:

#### Closes #3309

**Comment by illfated** [comment]
I will post my personal PR change approval as soon as the critical typo is resolved.

**Review by eross-msft — APPROVED** [review]

**Comment by PRMerger12** [comment]
@eross-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**